README.md

```diff
@@ -1,8 +1,9 @@
 # Unofficial Patch 9.1f for Metro for Steam skin
 
-This unofficial patch for [Metro for Steam skin](https://metroforsteam.com) fixes bugs with current version of the skin, implements newly added Steam client features and adds optional ones for unhindered and customized design.
 
-Additional details and feedback are available on [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam community group. Installation instructions can be found on the [Unofficial Patch thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
+This unofficial patch for the [Metro for Steam skin](https://metroforsteam.com) fixes bugs with current version of the skin, implements newly added Steam client features and adds optional ones for productivity and aesthetic.
+
+Additional details and feedback are available on [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
```

Comment:
```suggestion
Additional details and feedback are available on [MetroSkin](https://steamcommunity.com/groups/metroskin) Steam community group. Installation instructions can be found on the [Steam discussion thread](https://steamcommunity.com/groups/metroskin/discussions/0/141136086931804907).
```

Reply:
where did "community" go - that was ur previous suggestion, already incorporated, lol. :)

Reply:
Personally don't think "community" is needed, but I've added a small correction in 1307ca1.